pkg/storaged/pages.jsx

```diff
@@ -591,7 +592,7 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                     <CardBody>
                         <Split hasGutter>
                             { icon && <SplitItem>{icon}</SplitItem> }
-                            <SplitItem isFilled><strong>{name}</strong>{info}</SplitItem>
+                            <SplitItem isFilled><strong><Truncate content={name} /></strong>{info}</SplitItem>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19919-20240202-142745-1d909c55-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)